suite-native/labeling/src/components/TransactionOutputLabelEditable.tsx

```diff
@@ -0,0 +1,62 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+    WithLabelingState,
+    selectOutputLabel,
+    updateOutputLabelThunk,
+} from '@suite-common/local-first-storage';
+import type { StaticSessionId } from '@trezor/connect';
+
+import { EditableLabelLayout } from './EditableLabelLayout';
+import { LabelEditForm } from './LabelEditForm';
+import { useIsLabelingEnabled } from './useIsLabelingEnabled';
+
+type TransactionOutputLabelEditableProps = {
+    txId: string;
+    outputIndex: number;
+    deviceStaticSessionId: StaticSessionId;
+};
+
+export const TransactionOutputLabelEditable = ({
+    txId,
+    outputIndex,
+    deviceStaticSessionId,
+}: TransactionOutputLabelEditableProps) => {
+    const isLabelingEnabled = useIsLabelingEnabled();
+    const dispatch = useDispatch();
+
+    const label =
+        useSelector((state: WithLabelingState) =>
+            selectOutputLabel({
+                state,
+                txId,
+                outputIndex,
+                deviceStaticSessionId,
+            }),
+        )?.label ?? null;
+
+    if (!isLabelingEnabled) {
+        return null;
+    }
+
+    return (
+        <EditableLabelLayout label={label}>
+            {({ onClose }) => (
+                <LabelEditForm
+                    label={label ?? ''}
```

Comment:
```suggestion
                    label={label}
```

and return '' from selector in case that label do not exists.

Reply:
I kinda don't like this. I designed the labeling (evolu) in a way, that `<no label>` is represented by a `null`. There is explicit conversion (see the `suite-common/local-first-storage/src/labeling/normalizeLabel.ts`).

Here I need to put a default value into "HTML" `input` so this is IMO the correct place to convert `null` (no label) to `''` for the input component.

Reply:
ok, lets keep it as it is.